zerver/tests/test_slack_importer.py

```diff
@@ -32,9 +32,14 @@
 import json
 
 import os
-import mock
+from mock import mock, patch
 from typing import Any, AnyStr, Dict, List, Optional, Set, Tuple, Text
 
+class MuteFunction():
+    # A mock class to temporarily suppress output to stdout
+    def write(self, s: str) -> None:
+        pass
+
```

Comment:
So, this works, but it's a sorta custom hack.

The normal way to do this sort of suppression is (1) to do the printing via the `logging` module and (2) to either mock out `logging` or often better, just set a high log-level for the test suite.  You can see an example of this in the `zulip.send_email` logger that we use.